### PR TITLE
Redesign/103 post creation page re design

### PR DESCRIPTION
**Pull request by josephchimebuka**
I redesigned the post creation page. Addressing the issue #108

### CODE REVIEW

File: JoyboyCommunity/src/modules/post/index.tsx [hunk 1]
```diff
@@ -50,28 +50,47 @@ export default function CreatePost() {
...
+       </TouchableOpacity>
+
+          <TouchableOpacity>
+           <Icons source={require('../../../assets/gif-02.png')} />
+       </TouchableOpacity>
+
+          <TouchableOpacity>
+           <Icons source={require('../../../assets/sticky-note-02.png')} />
+       </TouchableOpacity>
+      </IconContainer>
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/post/index.tsx:91]: can you fix the formatting for this part? running `yarn format` will automatically fix the formatting issues

File: JoyboyCommunity/src/modules/post/styled.ts [hunk 2]
```diff
@@ -4,25 +4,73 @@ import styled from 'styled-components/native';
...
-  width: 100%;
-  padding: 16px;
+
+  padding: 11px 16px;
+  margin: 0 16px;
+  display: flex;
+
+  padding: 11px 16px;
+  margin: 0 16px;
+  display: flex;
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/post/styled.ts:14]: Please remove the duplicates

File: JoyboyCommunity/src/modules/post/styled.ts [hunk 3]
```diff
@@ -4,25 +4,73 @@ import styled from 'styled-components/native';
...
+    height: 24px;
+    resize: 'contain';
+    margin-left: 10;
+`;
+
+export const Sendbutton = styled(Image)`
+    width: 56;
+    height: 56;
+    resize: 'contain';
+    margin-left: 10;
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/post/styled.ts:66]: There is no unit on margin-left

**Review by ugur-eren — CHANGES_REQUESTED** [review]
I've requested some changes. Also icons should not be png's, it should be svg. You can extract svg from the figma file, and you can use it by creating an icons folder in the assets. You can use svgr to convert svg's into react native svg.

**Comment by josephchimebuka** [comment]
Applied all the changes and formatted the code as well

File: JoyboyCommunity/src/modules/post/index.tsx [hunk 4]
```diff
@@ -52,28 +67,53 @@ export default function CreatePost() {
...
+          </Typography>
         </Pressable>
 
         <PostButton onPress={handlePostNoteFn} disabled={isCreateDisabled}>
-          <Typography variant="ts15r">Post</Typography>
-        </PostButton>
+          <Typography style={{color: '#4B799F', fontSize: 17}} variant="ts15r">
+            Draft
+          </Typography>
       </Container>
```
Comment (ugur-eren) [inline JoyboyCommunity/src/modules/post/index.tsx:79]: PostButton doesn't have a closing tag

File: JoyboyCommunity/assets/svgs/svgComponents/Gif.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Svg, { Path } from 'react-native-svg';
+
+function GifIcon(props) {
```
Comment (ugur-eren) [inline JoyboyCommunity/assets/svgs/svgComponents/Gif.tsx:4]: For every svg, can we have typescript types too? For example:
```tsx
export const GifIcon: React.FC<SvgProps> = (props) => {
  ...
}
```

**Comment by josephchimebuka** [comment]
I have removed the unused components and images

**Pull request merged**